helm/aws-load-balancer-controller/values.yaml

```diff
@@ -319,3 +319,8 @@ clusterSecretsPermissions:
   # allowAllSecrets allows the controller to access all secrets in the cluster.
   # This is to get backwards compatible behavior, but *NOT* recommended for security reasons
   allowAllSecrets: false
+
+# whether or not this ingressclass is the default ingressclass inside the cluster
+controller:
```

Comment:
lets have in this structure
```
# ingressClassConfig contains configurations specific to the ingress class
ingressClassConfig:
    default: true
```
We will group the remaining ingress class configurations under this later.